libsovtoken/src/api/mod.rs

```diff
@@ -81,7 +81,7 @@ pub extern "C" fn create_payment_address_handler(
     let handler = CreatePaymentHandler::new(CryptoSdk {});
     let ec = handler.create_payment_address_async(wallet_handle, config, payment_closure);
     trace!("api::create_payment_address_handler << result: {:?}", ec);
-    return ec as i32;
+    ec as i32
 }
```

Comment:
we want to leave these returns for now.   the team that started libsovtoken made the decision to always use return, even though rust allows the syntax to not use it

Reply:
Sounds good. I just wanted to get clippy's output to a reasonable level, but if it'll cause more problems, let's not worry about if for now.